etc/profile-a-l/cointop.profile

```diff
@@ -0,0 +1,50 @@
+# Firejail profile for cointop
+# This file is overwritten after every install/update
+# Persistent local customizations
+include cointop.local
+# Persistent global definitions
+include globals.local
+
+noblacklist ${HOME}/.config/cointop
+
+include disable-common.inc
+include disable-devel.inc
+include disable-exec.inc
+include disable-interpreters.inc
+include disable-proc.inc
+include disable-programs.inc
+include disable-shell.inc
+include disable-write-mnt.inc
+include disable-X11.inc
+include disable-xdg.inc
+
+apparmor	# if you have AppArmor running, try this one!
+caps.drop all
+ipc-namespace
+netfilter
+no3d
+nodvd
+nogroups
+noinput
+nonewprivs
+noroot
+nosound
+notv
+nou2f
+novideo
+protocol inet,inet6
+seccomp
+shell none
+tracelog
+
+disable-mnt
+private-cache
```

Comment:
`private-bin`?

Reply:
Will need to build a new version of firejail to test this - but sure.

Reply:
WHich version do you use?

Reply:
I have 0.9.66 installed; but give me a bit.  I'll upgrade to the rc and try it out there.